src/main/java/io/javalin/Javalin.java

```diff
@@ -202,6 +200,33 @@ public Javalin defaultCharacterEncoding(String characterEncoding) {
         return this;
     }
 
+    /**
+     * Sets maximum body size allowed to cache in request. Requests with [Transfer-Encoding: chunked] are not cached.
+     *
+     * The method must be called before {@link Javalin#start()}.
+     */
+    public Javalin maxBodySizeForRequestCache(long value) {
+        ensureActionIsPerformedBeforeServerStart("Changing request cache body size");
+        this.maxRequestCacheBodySize = value;
+        return this;
+    }
+
+    /**
+     * Disables request body caching.
+     *
+     * The method must be called before {@link Javalin#start()}.
+     */
+    public Javalin disableRequestBodyCache() {
+        return maxBodySizeForRequestCache(0);
+    }
+
+    /**
+     * @return maximum body size allowed to cache in request. Requests with [Transfer-Encoding: chunked] are not cached.
+     */
+    public long maxBodySizeForRequestCache() {
```

Comment:
Unnecessary getter?

Reply:
I suppose it could be left here, the same way as we have setters-getters for other configuration fields.